tools/swift-plugin-server/allow-jit.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+<key>com.apple.security.cs.allow-jit</key>
```

Comment:
Necessary in order for JSC to expose `globalThis.WebAssembly` — I don't think there's any security risk here because the same surface area is exposed via `/System/Library/Frameworks/JavaScriptCore.framework/Versions/Current/Helpers/jsc`.